### ISSUE TITLE
baseBCount incorrect, while baseACount is not

### ISSUE THREAD

**Issue by therats-collab** [therats-collab/worm-generator#4]
In the code listed below, baseACount has a final value of 12 (correct), but baseBCount has a final value of 24. In order to finish #1 , this bug needs to be fixed.

```python
for filename in os.listdir(basePath):
    if filename.startswith("baseA"):
        baseACount = baseACount + 1
        print(baseACount)

for i in os.listdir(basePath):
    if filename.startswith("baseB"):
        baseBCount = baseBCount + 1
        print(baseBCount)
```

**Comment by therats-collab** [comment]
ON3 OF TH3M 1S  "for filename in..." TH3 OTH3R "for **i** in..." 1 TH1NK TH4T COULD B3 TH3 SOURC3 OF TH3 BUG. --T3R3Z1

**Comment by therats-collab** [comment]
That was precisely the issue. Fixed in v0.1.1.
